staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -188,6 +188,15 @@ var (
 		},
 		[]string{"requestKind"},
 	)
+
+	requestErrorsTotal = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_request_errors_total",
+			Help:           "Number of requests which have resulted in an apiserver response error.",
```

Comment:
Maybe something like:

Number of requests which apiserver aborted in self-defense.
Number of requests which apiserver terminated in self-defense.
Number of requests which apiserver terminated due to lack of server resources.

Reply:
I guess I'll vote for "terminated" as the verb since "termination" works better in names.

Reply:
Opted for #2